### DISCUSSION TITLE
File streaming

### DISCUSSION THREAD

**Discussion by rogerbl** [doitandbedone/ispyagentdvr-docker#510] [General]
I tried to connect to a mp4 file but got this error in the log:

23:14:31 | Etneelva: DoStart:    at CoreLogic.Sources.Combined.MediaStream.DoStart()
-- | --
23:14:31 | Error: Etneelva: DoStart:    at CoreLogic.Sources.Combined.MediaStream.DoStart()
23:14:31 | Reader: Etneelva: OPEN_INPUT:Protocol not found

What can be wrong?

**Comment by ispysoftware** [comment]
what source type are you using and what path?
